src/Parser/Internal/AbstractCommandDocBlockParser.php

```diff
@@ -169,6 +170,14 @@ protected function processAliases($tag)
     }
 
     /**
+     * Process the comma-separated list of commands to call.
+     */
+    protected function processCalls($tag)
+    {
+        $this->commandInfo->setCalls((string)$tag->getDescription());
```

Comment:
If there are multiple @calls annotations, this function will be called multiple times. Currently, this implementation overwrites via `setCalls()`. It could also  call getCalls() first and merge. Might want to move where the csv is converted to an array here, maybe.